docs/source/index.rst

```diff
@@ -54,6 +54,7 @@ Use the clouds **easily** and **cost effectively**, without needing cloud infra
    reference/auto-stop
    examples/spot-jobs
    reference/benchmark/index
+   reference/tpu
```

Comment:
Move to `USER GUIDES`? WDYT?

Reply:
Either seems good; my vote would be here, as it's a distinguishing feature that deserves to be featured higher ;)

Reply:
Got it. Makes sense.